onnx2torch/node_converters/roialign.py

```diff
@@ -91,6 +142,30 @@ def symbolic(graph: torch_C.Graph, *args) -> torch_C.Value:
 @add_converter(operation_type='RoiAlign', version=10)
 def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint: disable=unused-argument
     node_attributes = node.attributes
+    coordinate_transformation_mode = CTMAttribute.OUTPUT_HALF_PIXEL
+    mode = node_attributes.get('mode', 'avg')
+    output_height = node_attributes.get('output_height', 1)
+    output_width = node_attributes.get('output_width', 1)
+    sampling_ratio = node_attributes.get('sampling_ratio', 0)
+    spatial_scale = node_attributes.get('spatial_scale', 1.0)
+
+    return OperationConverterResult(
+        torch_module=OnnxRoiAlign(
+            coordinate_transformation_mode=coordinate_transformation_mode,
+            mode=mode,
+            output_height=output_height,
+            output_width=output_width,
+            sampling_ratio=sampling_ratio,
+            spatial_scale=spatial_scale,
+        ),
+        onnx_mapping=onnx_mapping_from_node(node),
+    )
+
+
+@add_converter(operation_type='RoiAlign', version=16)
```

Comment:
why do you separate 10 form 16 ? They are almost the same, except 'coordinate_transformation_mode' attribute, but you use get method.

Reply:
due to different default values for CTMAttr